third_party/systemlibs/syslibs_configure.bzl

```diff
@@ -44,6 +43,7 @@ VALID_LIBS = [
     "snappy",
     "tblib_archive",
     "termcolor_archive",
+    "typing_extensions_archive",
```

Comment:
Can you remove this line from this commit, its technically broken here since the BUILD file is missing. I fixed and added it in: https://github.com/tensorflow/tensorflow/pull/44542

I've been on the fence about having this allow-list of working syslibs since its a maintenance burden but I think cases like typing_extensions is good to have so that its only added when its tested and working. but on the other hand googleapis is not so good :/ Your script is great tho so should help :)

Reply:
Will do.

I do think it is a good thing to have IFF coupled with a (CI) test that the list is up to date. It allows to catch changed names (that happened already a couple of times) and removed or added libs from the user site